crypto.c

```diff
@@ -444,6 +444,7 @@ int dtls_ecdh_pre_master_secret(unsigned char *priv_key,
   uint32_t result_x[8];
   uint32_t result_y[8];
 
+  assert(key_size == 8);
```

Comment:
I understand `dtls_ec_key_to_uint32` and `dtls_ec_key_from_uint32` to take the key size in bytes. Therefore, I would expect `key_size` to be `8 * size_of(uint32_t)` or `sizeof(priv_key) == sizeof(pub_key_x) = ...`.

Reply:
```
./dtls-client -c TLS_ECDHE_ECDSA_WITH_AES_128_CCM localhost
dtls-client: /home/achim/repos/tinydtls/crypto.c:447: dtls_ecdh_pre_master_secret: Assertion `key_size == 8' failed.
Abgebrochen (Speicherabzug geschrieben)
```

Looks like @obgm  is right.

```
assert(key_size == sizeof(priv));
```

works.

Reply:
Yes, now fixed.